### ISSUE TITLE
Can this be used in production?

### ISSUE THREAD

**Issue by levrik** [aweary/react-copy-write#58]

**Comment by wKovacs64** [comment]
Relevant tweet (second part) from the author: https://twitter.com/aweary/status/1049352654315347968

We'll see if anything changes in 10 days. 😉

**Comment by levrik** [comment]
Oh. That looks interesting! Thanks for pointing me to this tweet.

**Comment by sykesd** [comment]
It's been nearly 2 months since those tweets, and about a month since the React Conf mentioned. 

Are we to understand that the proposed "hooks" addition to React takes over from this library?

Just wondering. I created a very simple app using this library, and really liked how it turned out. But before investing more time using this library, I was wondering whether or not its creator is considering further development?

Basically, I have the same questions (still) as the OP.

**Comment by aweary** [comment]
Hey y'all, sorry for ghosting you. I'm currently on paternity leave so I'm not focusing much on this project.

I wanted everyone to wait for ReactConf and hooks because I didn't feel comfortable recommending react-copy-write when I knew there was a big API change coming up. I wanted to make sure that everyone had the full picture before making any recommendations. I'd hate for you to adopt RCW and then regret it because you didn't know hooks were coming in less than 2 weeks 🙂

After more thought I think there's still a solid use case for RCW and I definitely want to continue supporting it and developing it!
